parse.go

```diff
@@ -624,9 +624,16 @@ func (p *parser) parseEvents(pos *Pos, n *yaml.Node) []Event {
 }
 
 // https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#permissions
-func (p *parser) parsePermissions(pos *Pos, n *yaml.Node) *Permissions {
+func (p *parser) parsePermissions(pos *Pos, n *yaml.Node, config *Config) *Permissions {
 	ret := &Permissions{Pos: pos}
 
+	if config != nil && config.EnforceEmptyPermissions && pos.IsTopLevel() {
+		if !isNull(n) {
+			p.errorf(n, "expected empty Permissions. If this is a mistake, please disable "+
+				"enforce_empty_permissions in the config file.")
```

Comment:
Some teams use `read-all` as the default, or `contents`, instead of empty.

What do you think about an option to make sure that
- default permissions are reduced in the beginning of a workflow
- default permissions are read-only

On the other hand, maybe it's also a good idea to go with one standard instead of having multiple ways in the community.

What are your thoughts on this @rhysd?

Reply:
I like the idea of enforcing no permissions. This would raise security awareness in areas of GitHub Actions which are not obvious.

Reply:
Without setting the var in the config the behaviour is the same as we currently have, which I believe is the standard, as GitHub docs don't force setting any particular permissions.
This option is mostly concerned for those who want to follow the security hardening guidelines. 

> Some teams use read-all as the default, or contents, instead of empty.

Regarding this aspect, perhaps I can tune the config, and add a couple of options. For example, instead of just `enforce_empty_permissions`, something like `enforce_permissions` with 2 options read-only and empty instead of the current boolean.

> On the other hand, maybe it's also a good idea to go with one standard instead of having multiple ways in the community.

This one might require more discussion with @rhysd. I believe we should have a standard, but also allow the user to configure the linter to their needs. 
Should we go with the most secure options right of the bat and have the users disable those options which are too secure?